src/feel-free-to-skip-ahead.md

```diff
@@ -0,0 +1,27 @@
+# Feel free to skip ahead!
+
+We're about to go on a journey together. However, there's no single way to
+travel. You have two options here:
+
+* Learn about some assembly code and exactly how things boot.
+* Ignore all that and head straight into Rust.
+
+Both ways are totally valid. Which way is "right" is up to you. If you'd like
+to write each and every line of code yourself, then move on to the next
+chapter. But if you're more excited about writing Rust code than dealing with
+the arcane details of the `x86` platform, don't worry about skipping ahead to
+"[A simple VGA driver]". The chapters up until then will be getting us from
+nothing to a main function written in Rust, and this involves a bunch of
+assembly programming that you may or may not find interesting. In fact, it's so
+common that there's a convention in the hobbyist OS development community to
+have a project called "bare bones" that sets up all of this stuff. So if you're
+not interested in the assembly part, you can skip ahead, and not miss out on
+a whole lot. The chapters up until then will be building up the bare bones, it's
+the chapters after that that dive into more interesting things.
```

Comment:
I'm not sure "more interesting" things should be used here. Learning `x86` can be super cool and fun (you showed me that) and you're being way too negative about it in this text!